Decred Wallet/Features/More/More Features/Security Tools/Validate Addresses/ValidateAddressesViewController.swift

```diff
@@ -146,7 +146,8 @@ class ValidateAddressesViewController: UIViewController, UITextViewDelegate, Flo
         if capturedText.starts(with: "decred:") {
             capturedText = capturedText.replacingOccurrences(of: "decred:", with: "")
         }
+        self.addressText.textViewDidBeginEditing(self.addressText)
         self.addressText.text = capturedText
-        self.validateAdd(address: capturedText)
+        self.toggleValidateButtonState(addressHasText: self.addressText.text)
```

Comment:
A cleaner alternative would be to use `self.addressText.setText()` which simulates keyboard input.